src/EnergyPlus/DataGlobalConstants.cc

```diff
@@ -129,6 +129,9 @@ ResourceType AssignResourceTypeNum(std::string const &ResourceTypeChar)
         } else if (SELECT_CASE_var == "DISTRICTHEATING") {
```

Comment:
Use `getEnumerationValue` instead of a custom function.

Reply:
I'll eliminate these functions and use `getEnueratationVaule` and `std::array string_view`s on a separate branch for enum.